src/POData/ObjectModel/ODataLink.php

```diff
@@ -55,6 +55,26 @@ class ODataLink
      */
     public $expandResult;
 
+    /**
+     * ODataLink constructor.
+     * @param string $name
+     * @param string $title
+     * @param string $type
+     * @param string $url
+     * @param bool $isCollection
+     * @param ODataEntry|ODataFeed $expandedResult
+     * @param bool $isExpanded
+     * @param null $expandResult
+     */
+    public function __construct($name = null, $title = null, $type = null, $url = null, $isCollection = null)
```

Comment:
If these parms are nullable, please update the doc block to match - ie, string|null